OpenOversight/app/utils.py

```diff
@@ -559,20 +560,18 @@ def find_date_taken(pimage):
     if isinstance(pimage, PngImageFile):
         return None
 
-    if pimage._getexif():
+    exif = hasattr(pimage, '_getexif') and pimage._getexif()
+    if exif:
         # 36867 in the exif tags holds the date and the original image was taken https://www.awaresystems.be/imaging/tiff/tifftags/privateifd/exif.html
```

Comment:
In my last job, I worked a lot with [image formats](https://github.com/AllenCellModeling/aicsimageio)....this shit tilts me like no other :upside_down_face: